test/signs.cc

```diff
@@ -243,6 +260,71 @@ TEST(Signs, TestGetGuideTowardString_roundabout_toward_A1) {
   TryGetGuideTowardString(signs, 1, true, "A 1");
 }
 
+TEST(Signs, TestGetGuideString_BranchOnly) {
+  // Create guide sign
+  // Specify input in descending consecutive count order
+  Signs signs =
+      GetGuideSigns({std::make_tuple("US 322 West", 1, 1), std::make_tuple("US 22 West", 1, 0),
```

Comment:
Can we have a test case where the consecutive count is > 1, say 2 of them have `2` and the result shows both items getting selected when `limit_by_consecutive_count` is set (for different values of. `max_count`)?

Reply:
@mandeepsandhu pushed up more tests
Added more TestGetGuideString tests with no, 1, and multiple consecutive counts